src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
```

Comment:
@cerinman I thought about this some more. Let's say we have numbers like `35156.25` and `35352.25`, 
`Math.ceil(d[axis] / 1000) * 1000` returns `36000` for both of them. As a result, the chart will be much smoother. But I'm not sure if that is what you want. If you want the chart more accurate, I would do this to increase the value just a little in relative to its current value, so it wouldn't be too close to the top and bottom
```javascript
return d[axis]*1.001
```